plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlCaseInsensitiveMapping.java

```diff
@@ -71,4 +71,10 @@ protected String quoted(String name)
         name = name.replace(identifierQuote, identifierQuote + identifierQuote);
         return identifierQuote + name + identifierQuote;
     }
+
+    @Test
+    public void forceTestNgToRespectSingleThreaded()
+    {
+        // do nothing
```

Comment:
please a link to comment that explained why this is needed, also create TODO with an issue in github to remove that whenever we update testng 7.4.0+

Reply:
Thanks for your comments, filed https://github.com/trinodb/trino/issues/8571